.vscode/settings.json

```diff
@@ -1,4 +1,3 @@
 {
```

Comment:
do you mean to check in this change?

Reply:
Maybe it shouldn't be part of this change, but in general I have to do this to rules_nodejs everytime I try to edit the repo in vscode. It breaks the bazel vscode extensions ability to format build files